packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -638,14 +652,20 @@ export class ARRenderer extends EventDispatcher {
       this.lastAngle = angle;
     } else if (this.isTranslating) {
       fingers.forEach(finger => {
-        if (finger.inputSource !== this.inputSource ||
-            finger.results.length < 1) {
+        if (finger.inputSource !== this.inputSource) {
           return;
         }
 
-        const hit = this.getHitPoint(finger.results[0]);
```

Comment:
I think this would work
```
        const hit = finger.results.length >= 1 ? this.getHitPoint(finger.results[0]) : this.getTouchLocation();
        if (hit == null) {
          return;
        }
```

Reply:
Ooh, I like that.